myenv2.yaml

```diff
@@ -0,0 +1,34 @@
+# Conda environment specification. The dependencies defined in this file will
```

Comment:
I think it would be nicer if we generate the conda yaml on the fly instead of upload it. For it you can execute 'scripts/generate_conda_file.py'. There are several ways to do this, in a notebook you can use the magic command `%run`:
```
%run ./scripts/generate_conda_file.py
```

Reply:
Using generate_conda_file instead now